modules/registry-configuring-storage-aws-upi.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * architecture/registry
+//* architecture/installation-.adoc
+
+[id="registry-configuring-storage-aws-upi-{context}"]
+= Configuring registry storage for AWS with UPI
+
+During installation, your cloud credentials are sufficient to create a S3 bucket
+and the Registry Operator will automatically configure storage.
+
+If the Registry Operator cannot create an S3 bucket and automatically configure
+storage, you can create a S3 bucket and configure storage with the following
+procedure.
+
+.Prerequisites
+
+* A cluster on AWS with upi
+
+.Procedure
+
+. Create a S3 bucket:
++
+----
+$ oc create configs.imageregistry.operator.openshift.io
+
+storage:
+  s3:
+    bucket:
+    region: <region-name>
+----
+
+. Set up a link:https://docs.aws.amazon.com/AmazonS3/latest/dev/mpuoverview.html#mpu-abort-incomplete-mpu-lifecycle-config[Bucket Lifecycle Policy]
+to abort incomplete multipart uploads that are one day old.
+
+. Fill in the storage configuration in
+`configs.imageregistry.operator.openshift.io/cluster`:
```

Comment:
If you delete all s3 part on AWS with UPI, it will be generated automatically, admin doesn't need to input manually.

Reply:
Admin may want to create a cluster with credentials that doesn't allow to create buckets. In this case the administrator should configure it manually.